src/main/webapp/app/exercises/shared/statistics/exercise-management-statistics-dto.ts

```diff
@@ -0,0 +1,17 @@
+export class ExerciseManagementStatisticsDto {
+    // average score
+    averageScoreOfExercise: number;
+    maxPointsOfExercise: number;
+    scoreDistribution: number[];
+    numberOfExerciseScores: number;
+
+    // participation rate
+    numberOfParticipations: number;
+    numberOfStudentsInCourse: number;
+
+    // questions
+    numberOfQuestions: number;
+    numberOfAnsweredQuestions: number;
+
+    constructor() {}
```

Comment:
I think this is the default value so you can omit this.

Reply:
👍